apps/web/src/pages/api/decode.ts

```diff
@@ -21,7 +21,10 @@ export default async function handler(req: NextApiRequest, res: NextApiResponse)
     const decodeResult = decodeFunctionData({ abi: JSON.parse(abi), data: calldata })
     const functionInfo = getAbiItem({
       abi: JSON.parse(abi),
-      name: decodeResult.functionName,
+      name:
+        decodeResult.functionName !== 'release'
+          ? decodeResult.functionName
+          : '0x37bdc99b', // manually set function signature for release(_milestone) instead of release()
```

Comment:
why replace this ?

Reply:
otherwise it picks up `release()` instead of the `release(_milestone)`